src/components/MeasurementDetail.vue

```diff
@@ -32,28 +39,39 @@
   </v-dialog>
 </template>
 
-<script>
+<script >
 //import {useMeasurementStore} from "@/store/MeasurementStore";
 //import {mapStores} from "pinia";
 
+import Config from "@/config";
+
 export default {
   name: "MeasurementDetail",
 
   data() {
-    return {}
+    return {
+      opened: false,
+    }
   },
 
   props: {
-    measurement: Object
+    measurement: Object,
   },
 
   computed: {
     //...mapStores(useMeasurementStore)
   },
 
+  created() {
+  },
```

Comment:
Tady taky prosím promazat zakomentované a nevyužité věci

Reply:
hotovo